codeworld-api/codeworld-api.cabal

```diff
@@ -24,12 +24,12 @@ Library
   Other-modules:       CodeWorld.Color,
                        CodeWorld.Picture,
                        CodeWorld.Event,
-                       CodeWorld.Prediction,
                        CodeWorld.Driver
   Build-depends:       base < 5,
                        cereal,
                        cereal-text,
                        codeworld-game-api,
+                       codeworld-prediction,
```

Comment:
Do you want to call this package codeworld-prediction?  If it's intended to be useful outside of CodeWorld, it may be worth its own top-level package name.  But you're welcome to start it out here, and later once it's stabilized, you can fork to an outside package, and modify codeworld-api to use that instead.